### DISCUSSION TITLE
From Terminal, unable to use sudo (file system with the 'nosuid' option set)

### DISCUSSION THREAD

**Discussion by pihomeserver** [coder/code-server#6443] [General]
Hello
Self hosting code-server on a Linux Debian VPS it works well for my needs except from a terminal when i want to run a command using sudo i get an error

![image](https://github.com/coder/code-server/assets/3817578/363bb638-4cb0-42f0-bdeb-3263ec6835ed)

Any idea how to configure in order to get an access to the command or if it's a feature for any security reason ?

**Comment by code-asher** [comment]
Sounds like the ownership on sudo is wrong.  How are you putting 
the VPS together?

It should look something like this:

```
$ ls -la $(which sudo) 
-rwsr-xr-x 1 root root 232416 Apr  3 18:00 /usr/bin/sudo 
```
